internal/metrics/otelmetrics/factory.go

```diff
@@ -131,3 +131,10 @@ func attributeSetOption(tags map[string]string) metric.MeasurementOption {
 	}
 	return metric.WithAttributes(attributes...)
 }
+
+func counterNamingConvention(name string) string {
+	if !strings.HasSuffix(name, "_total") {
+		name += "_total"
```

Comment:
did we have a precedent of doing this elsewhere in Jaeger?

Reply:
Yes, this was being done in prometheus metrics lib.